DAAppsViewController/DAAppViewCell.m

```diff
@@ -42,7 +61,12 @@ + (void)initialize
         
         NSInteger numberOfStars = 11;
         NSMutableArray *starRatingImages = [[NSMutableArray alloc] initWithCapacity:numberOfStars];
-        UIImage *starsImageSheet = [UIImage imageNamed:@"DAAppsViewController.bundle/DAStarsImage"];
+        UIImage *starsImageSheet = [UIImage imageNamedFromMainBundleOrFramework:@"DAAppsViewController.bundle/DAStarsImage"];
+        if (!starsImageSheet) {
```

Comment:
This nil check is redundant as it's taken care of in the category method `imageNamedFromMainBundleOrFramework` (assume it was accidentally left after refactoring).